squire_lib/src/tournament.rs

```diff
@@ -961,6 +961,116 @@ impl Tournament {
 }
 
 impl TournamentSeed {
+    const MAGIC_SYNONYMS: &[&'static str] = &[
+        "Magic",
+        "Illusion",
+        "Wizardry",
+        "Alchemy",
+        "Bewitchment",
+        "Conjuring",
+        "Devilry",
+        "Divination",
+        "Enchantment",
+        "Exorcism",
+        "Horoscopy",
+        "Incantation",
+        "Legerdemain",
+        "Magnetism",
+        "Necromancy",
+        "Occultism",
+        "Predicition",
+        "Presage",
+        "Prestidigitation",
+        "Prophecy",
+        "Soothsaying",
+        "Sorcery",
+        "Instant",
+        "Superstition",
+        "Thaumaturgy",
+        "Voodoo",
+        "Witchcraft",
+    ];
+
+    const GATHERING_SYNONYMS: &[&'static str] = &[
+        "Gathering",
+        "Association",
+        "Caucus",
+        "Collection",
+        "Conclave",
+        "Conference",
+        "Congregation",
+        "Convention",
+        "Crowd",
+        "Function",
+        "Group",
+        "Mass",
+        "Meeting",
+        "Party",
+        "Throng",
+        "Acquisition",
+        "Affair",
+        "Aggregate",
+        "Aggregation",
+        "Band",
+        "Body",
+        "Bunch",
+        "Clambake",
+        "Company",
+        "Concentration",
+        "Concourse",
+        "Congress",
+        "Convocation",
+        "Crush",
+        "Drove",
+        "Flock",
+        "Gain",
+        "Heap",
+        "Herd",
+        "Horde",
+        "Huddle",
+        "Junction",
+        "Knot",
+        "Levy",
+        "Meet",
+        "Muster",
+        "Parley",
+        "Pile",
+        "Rally",
+        "Roundup",
+        "Society",
+        "Stock",
+        "Stockpile",
+        "Swarm",
+        "Turnout",
+        "Union",
+        "Get-together",
+        "Social function",
+    ];
+
+    /// Returns an autogenerated tournament name
+    pub fn default_name() -> String {
+        let mut bytes = [0; 8];
+        let _ = getrandom::getrandom(&mut bytes); // should squire_lib be the source of randomness?
```

Comment:
> `// should squire_lib be the source of randomness?`
`getrandom` makes sys calls to get the entropy for "randomness". This is why we have to enable the `js` feature if we are targeting WASM.